File: app/src/main/java/com/example/bahroel/crudmahasiswa/Api/ApiMahasiswa.java
package com.example.bahroel.crudmahasiswa.Api;

public class ApiMahasiswa {
    public static final String BASE_URL = "http://192.168.8.101/crud_mahasiswa/api/apiMahasiswa.php?apicall=";

    public static final String URL_C_MHS = BASE_URL + "create_mahasiswa";
    public static final String URL_R_MHS = BASE_URL + "get_mahasiswa";
    public static final String URL_U_MHS = BASE_URL + "update_mahasiswa";
    public static final String URL_D_MHS = BASE_URL + "delete_mahasiswa&nrp=";
}

File: CRUD_Android/api/ApiMahasiswa.php
<?php 
	require_once "../config/Connection.php";
	require_once "../model/Mahasiswa.php";

	$api = $_GET['api'];
	$connection = new Connection();
	$mahasiswa = new Mahasiswa();

	if($_SERVER['REQUEST_METHOD'] == 'POST'){
		switch($api){
			case 'createMahasiswa' :
				$mahasiswa->createMahasiswa([
					'nrp' 		=>$_POST['nrp'],
					'nama'		=>$_POST['nama'],
					'jurusan' 	=>$_POST['jurusan'],
					'kelas'		=>$_POST['kelas'],
					'telp' 		=>$_POST['telp'],
					'alamat'	=>$_POST['alamat']
				]);
				break;
			case 'updateMahasiswa' :
				$mahasiswa->updateMahasiswa([
					'nrp' 		=>$_POST['nrp'],
					'nama'		=>$_POST['nama'],
					'jurusan' 	=>$_POST['jurusan'],
					'kelas'		=>$_POST['kelas'],
					'telp' 		=>$_POST['telp'],
					'alamat'	=>$_POST['alamat'],
				]);
				break;
			case 'deleteMahasiswa' :
				$mahasiswa->deleteMahasiswa($_POST['nrp']);
				break;
			// case 'getMahasiswa' :
			// 	$mahasiswa->getMahasiswa();
			// 	break;


		}
	}else{
			$mahasiswa->getDataMahasiswa();
	}
 ?>




